pyart/retrieve/test_kdp_proc.py

```diff
@@ -0,0 +1,69 @@
+""" Unit tests for pyart.retrieve.kdp_proc module. """
```

Comment:
I removed the file as it was basically a duplicate of the same file in the tests directory